srcpkgs/qbittorrent/template

```diff
@@ -19,38 +16,31 @@ distfiles="${SOURCEFORGE_SITE}/qbittorrent/qbittorrent-${version}.tar.xz"
 checksum=89cd79f58af4db346a9744e4bf61181c4bd40cce201b79a9f54ac31a8676e921
 CXXFLAGS=-std=gnu++17
 
-do_extract() {
-	local n=${pkgname}-${version}
-	mkdir -p build-nox build-x11
-	vsrcextract -C build-x11 ${n}.tar.xz
-	vsrcextract -C build-nox ${n}.tar.xz
+post_configure() { # qbittorrent-nox
+	configure_args="${configure_args//DGUI=ON/DGUI=OFF}"
+	configure_args="${configure_args//DWEBUI=OFF/DWEBUI=ON}"
+	(
+		cmake_builddir="nox-build"
+		do_configure
+	)
 }
 
-do_configure() {
-	cd ${wrksrc}/build-x11
-	./bootstrap.sh
-	QT_QMAKE=${XBPS_WRAPPERDIR} ./configure ${configure_args} --enable-gui --disable-webui
-	cd ${wrksrc}/build-nox
-	./bootstrap.sh
-	QT_QMAKE=${XBPS_WRAPPERDIR} ./configure ${configure_args} --disable-gui --enable-webui
+post_build() { # qbittorrent-nox
+	(
+		DESTDIR="${DESTDIR/qbittorrent/qbittorrent-nox}"
```

Comment:
don't think DESTDIR needs to be set for build

Reply:
done